scripts/jenkins-yoctobuild-build.sh

```diff
@@ -398,8 +398,19 @@ if grep mender_servers <<<"$JOB_BASE_NAME"; then
     # Use release tool to query for available docker names.
     for docker in $($WORKSPACE/integration/extra/release_tool.py --list docker -a ); do (
 
-        git=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker git)
-        docker_url=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker docker_url)
+        if $WORKSPACE/integration/extra/release_tool.py --help | grep -e '--map-name FROM-TYPE SERVICE TO-TYPE'; then
```

Comment:
Question: Is there any particular reason you are piping to grep instead of
```
if release_tool.py --map-name; then 
```
?

Reply:
There are sure different ways of doing this, I used this pipe to grep following the same logic that we detect version of `mender-artiafct` old (`-u`) or new (`-f`)

Your exact example will not work, as `release_tool.py --map-name` should return error even if the parameter exists, as it requires two additional positional arguments